exports/taskfiles/utils/remote.yaml

```diff
@@ -23,9 +23,8 @@ tasks:
     status:
       - >-
         diff
-        <(echo "{{.FILE_SHA256}}")
-        <(openssl dgst -sha256 "{{.OUTPUT_FILE}}"
-        | awk '{print $2}')
+          <(echo "{{.FILE_SHA256}}")
+          <(openssl dgst -sha256 "{{.OUTPUT_FILE}}" | awk '{print $2}')
```

Comment:
If we want to indent lines, we can't use `>-` since a newline + an indent gets interpreted verbatim.

On that note, this error caused the task to not be idempotent anymore. Perhaps we could add a test for idempotency by running it once, saving the file's modification date, then running it again to ensure the modification date didn't change?